src/apps/Gastro/components/TrafficOrder/TrafficOrderGastro.tsx

```diff
@@ -707,6 +707,8 @@ const TrafficOrder = ({ application }) => {
           beantragen/ erlangen.
         </p>
 
+        <FeeTable application={application} />
```

Comment:
In the spirit to keep the props footprint small, I would pass status and invoice_number direclty, since they are the only props needed for the component.